enlighten/DespikingFeature.py

```diff
@@ -0,0 +1,100 @@
+import logging
+from statistics import median
+
+import numpy as np
+
+from wasatch.ProcessedReading import ProcessedReading
+
+log = logging.getLogger(__name__)
+
+class DespikingFeature:
```

Comment:
In general, I would like at least a one-sentence class comment summarizing the purpose of each class.